src/map/handler/TouchGestures.js

```diff
@@ -105,19 +104,9 @@ L.Map.TouchGestures = L.Handler.extend({
         if (this._rotating) {
             var theta = Math.atan(vector.x / vector.y);
             var bearingDelta = (theta - this._startTheta) * L.DomUtil.RAD_TO_DEG;
-            if (inertia) {
-                if (!this._inertiaOvercomedAt && Math.abs(bearingDelta) >= inertia) {
-                    this._inertiaOvercomedAt = bearingDelta;
-                }
-            }
+
             if (vector.y < 0) { bearingDelta += 180; }
-            if (inertia) {
-                if (this._inertiaOvercomedAt) {
-                    bearingDelta -= this._inertiaOvercomedAt;
-                } else {
-                    bearingDelta = 0; 
-                }
-            }
+            if (inertia)      { bearingDelta = Math.abs(this._startBearing - bearingDelta) >= inertia ? (bearingDelta - this._startBearing) : 0; }
```

Comment:
@mauriciabad does this seem right to you?

Reply:
Nope, because I expect that once inertia is overcome, the map rotates freely until you let out. But with this change, if you overcome the inertia, and then rotate back; the map stops rotating untill you overcome the inertia again. There's like a dead zone.

Using `this._startBearing` is a good idea to reuse variables. But since the map has to remember if it has overcomed the inertia or not, it seems unavoidable to add another variable.